File: lab3/s1/ejb/src/main/java/ru/uvuv643/ejb/remote/dto/human/response/AllResponseDto.java
package ru.uvuv643.ejb.remote.dto.human.response;

import ru.uvuv643.ejb.remote.dto.human.HumanBeingDto;

import  jakarta.xml.bind.annotation.XmlAccessType;
import  jakarta.xml.bind.annotation.XmlAccessorType;
import  jakarta.xml.bind.annotation.XmlElementWrapper;
import jakarta.xml.bind.annotation.XmlRootElement;
import java.io.Serializable;
import java.util.List;

@XmlRootElement()
@XmlAccessorType(XmlAccessType.FIELD)
public class AllResponseDto implements Serializable {

        @XmlElementWrapper(name = "teamList")
        private List<HumanBeingDto> humanBeingDto;

        public AllResponseDto() {
                this(null);
        }

        public AllResponseDto(List<HumanBeingDto> humanBeingDto) {
                this.humanBeingDto = humanBeingDto;
        }

        public List<HumanBeingDto> getHumanBeingDto() {
                return humanBeingDto;
        }

        public void setHumanBeingDto(List<HumanBeingDto> humanBeingDto) {
                this.humanBeingDto = humanBeingDto;
        }
}

File: lab2/s1/src/main/kotlin/ru/uvuv643/soa/api/v1/dto/human/response/AllResponseDto.kt
package ru.uvuv643.soa.api.v1.dto.human.response

import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty
import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement
import io.swagger.v3.oas.annotations.media.Schema
import jakarta.xml.bind.annotation.XmlAccessType
import jakarta.xml.bind.annotation.XmlAccessorType
import jakarta.xml.bind.annotation.XmlElement
import jakarta.xml.bind.annotation.XmlElementWrapper
import jakarta.xml.bind.annotation.XmlRootElement
import ru.uvuv643.soa.api.v1.dto.human.HumanBeingDto

@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
data class AllResponseDto constructor(

        @XmlElementWrapper(name = "teamList")
        val humanBeingDto: List<HumanBeingDto>?,

        ) {
        constructor() : this(null)
}
